components/buttongroup/index.css

```diff
@@ -10,29 +10,17 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-.spectrum-ButtonGroup {
-  --spectrum-buttongroup-spacing-horizontal: var(--spectrum-spacing-300);
-  --spectrum-buttongroup-spacing-vertical: var(--spectrum-spacing-300);
-}
-
-.spectrum-ButtonGroup--sizeS {
+.spectrum-ButtonGroup,
+.spectrum-ButtonGroup--sizeM,
+.spectrum-ButtonGroup--sizeL,
+.spectrum-ButtonGroup--sizeXL {
```

Comment:
Could we remove these three classes, since only small is different than the default?

Reply:
yes, I've removed them and updated our docs and migration guide to reflect them being removed.